tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
```

Comment:
We now support `order` paramer, so please add it and order the pages as:
Java installation: order: 1
Build tools: order: 2
Java SIG: order: 3